lib/workers/bin/run_single_worker.rb

```diff
@@ -36,8 +38,8 @@
     exit
   end
 
-  opts.on("-f", "--force", "Ignore missing server roles and run anyway") do
-    options[:force] = true
+  opts.on("-r=ROLE", "--role=ROLE", "Set active role") do |val|
```

Comment:
"Set assigned role"

I think the master server still determines which assigned roles get activated.

Reply:
assigned and active are like status and state...

Reply:
oh, wait, you're assigning and activating the roles...
maybe you're ok as is...

Reply:
Yeah, my usecase is I want to run a single worker ("hey I hear there's a script for that") without MiqServer because I want to test something.

Without MiqServer there were no active roles so queue workers wouldn't do anything.